### PR TITLE
feat(Constants): added more error codes

### PR DESCRIPTION
**Pull request by kyranet**
**Please describe the changes this PR makes and why it should be merged:**

ref: https://github.com/discord/discord-api-docs/pull/3542

**Status and versioning classification:**

- Code changes have been tested against the Discord API, or there are no code changes
- I know how to update typings and have done so, or typings don't need updating
- This PR changes the library's interface (methods or parameters added)

### CODE REVIEW

**Review by vladfrangu — APPROVED** [review]
LGTM once upstream PR is merged - feel free to open one in -types too!

**Review by iCrawl — APPROVED** [review]

File: src/util/Constants.js
Comment (iCrawl) [inline src/util/Constants.js:668]:
```suggestion
 * * GUILD_NOT_AVAILABLE_IN_LOCATION
 * * GUILD_MONETIZATION_REQUIRED
```

**Pull request merged**
